opentracing-jaxrs2-itest/pom.xml

```diff
@@ -32,6 +32,10 @@
       <artifactId>junit</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Why noop dependency has been added to almost all modules?

Reply:
Maybe it's not required because some other dependencies bring it in as a transitive dependency, but since we depend on it (with imports from `io.opentracing.noop`), we should be explicit about the dependency.